### PR TITLE
register services

### PR DESCRIPTION
**Pull request by xalluna**
registered services and cleaned project of gitignored files

### CODE REVIEW

File: TcgPocket/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddSingleton(_configuration);
+        services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblyContaining<Startup>());
+        services.AddAutoMapper(typeof(IMapper));
+        services.AddValidatorsFromAssemblyContaining<IValidator>(ServiceLifetime.Scoped);
+        services.AddDbContext<DataContext>(options =>
+            options.UseSqlServer(_configuration.GetConnectionString(AppSettings.DefaultConnection)));
+    }
+}
```
Comment (gbrad777) [inline TcgPocket/Startup.cs:30]: add program config stuff to here and then have program make builder using this startup file

**Review by gbrad777 — CHANGES_REQUESTED** [review]

File: TcgPocket/Startup.cs [hunk 2]
```diff
@@ -10,20 +10,19 @@ public class Startup
 {
     private IConfigurationRoot _configuration { get; }
 
-    public Startup()
+    public Startup(WebApplicationBuilder builder)
     {
-        var builder = new ConfigurationBuilder()
-            .AddJsonFile("appsettings.json");
+        // builder;
```
Comment (gbrad777) [inline TcgPocket/Startup.cs:15]: does this need to be removed?

File: TcgPocket/Migrations/20230826051236_Initial.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (gbrad777) [inline TcgPocket/Migrations/20230826051236_Initial.cs:1]: was this initial migration needed?

Comment (xalluna) [reply TcgPocket/Migrations/20230826051236_Initial.cs:1]: I suppose it is not

File: TcgPocket/Startup.cs (hunk 2 above)
Comment (xalluna) [reply TcgPocket/Startup.cs:15]: I copied code from a console App I used with EF core. I had to do this manually, but `WebHostBuilder" does this automagically.

File: TcgPocket/Migrations/20230826051236_Initial.cs (hunk 3 above)
Comment (xalluna) [reply TcgPocket/Migrations/20230826051236_Initial.cs]: I didn't realize that migration '0' was there by default

**Review by gbrad777 — APPROVED** [review]

**Pull request merged**